puppet/lib/launchProcess.ts

```diff
@@ -109,6 +115,7 @@ export default async function launchProcess(
       } else {
         launchedProcess.kill('SIGKILL');
       }
+      if (dataDir) Fs.rmdirSync(dataDir, { recursive: true });
```

Comment:
I've noticed that rmdirSync throws errors every so often. I can't figure out why. We might want a try-catch block that retries x times.